integrations/extensions/starter-kits/watson-discovery/README.md

```diff
@@ -126,7 +135,7 @@ In the Actions global settings (which you get to from the gear icon in the upper
 
 If you look closely at the configurations in the actions that call the search extension ("Search", "Search within the mining topic", and "How many Bitcoins can exist?") you will notice that they set an assortment of parameters that impact how many bytes of data Watson Discovery returns.  Most notably, the `count` field is set to 3 (so we are asking for at most 3 search results) and the `return` field is set to `["title","metadata.source.url"]` so we are only getting the the title and URL from the document schema (in addition to the passages from the `text` field, because `passages.enabled` is `true` and `passages.fields` is `["text"]`).  If we omitted the `return` field, we would be getting back everything in the schema which in our demo example would include the full HTML of the FAQ page (which is fairly large).  In addition, we set `table_results.enabled` to `false` because we don't need tables in this application (but small tables may well be fine within the context of this limit).
```

Comment:
```suggestion
If you look closely at the configurations in the actions that call the search extension ("Search", "Search within the mining topic", and "How many Bitcoins can exist?") you will notice that they set an assortment of parameters that impact how many bytes of data Watson Discovery returns.  Most notably, the `count` field is set to 3 (so we are asking for at most 3 search results) and the `return` field is set to `["title","metadata.source.url"]` so we are only getting the title and URL from the document schema (in addition to the passages from the `text` field, because `passages.enabled` is `true` and `passages.fields` is `["text"]`).  If we omitted the `return` field, we would be getting back everything in the schema which in our demo example would include the full HTML of the FAQ page (which is fairly large).  In addition, we set `table_results.enabled` to `false` because we don't need tables in this application (but small tables may well be fine within the context of this limit).
```